python/s3-sns-sqs-lambda-chain/app.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+import os
+
+import aws_cdk as cdk
+
+from s3_sns_sqs_lambda_chain.s3_sns_sqs_lambda_chain_stack import S3SnsSqsLambdaChainStack
+
+
+app = cdk.App()
```

Comment:
```suggestion
import aws_cdk as cdk
from s3_sns_sqs_lambda_chain.s3_sns_sqs_lambda_chain_stack import S3SnsSqsLambdaChainStack

app = cdk.App()
```

Removed `os` since its not used. Removed some unnecessary spacing.